app/templates/administrative-units/administrative-unit/sites/index.hbs

```diff
@@ -30,7 +31,8 @@
           {{site.address.fullAddress}}
         </AuLink>
       </td>
-      <td>{{if (eq site.id this.model.primarySite.id) "Ja" "Nee"}}</td>
+      <td>{{site.siteTypes.label}}</td>
+      <td>{{if (eq site.id this.model.primarySite.id) "Ja" "NeeN"}}</td>
```

Comment:
Typo, should be "Nee" I think?

Reply:
I don't know, in the design it has been changed in Neen. Should I put 'Nee'?

Reply:
"Neen" is fine as well (although I think it sounds a bit weird), but then it shouldn't be a capital N :smile: